sdk/healthdataaiservices/azure-health-deidentification/src/samples/java/com/azure/health/deidentification/AsyncHelloWorld.java

```diff
@@ -24,10 +22,8 @@ public static void main(String[] args) {
         String inputText = "Hello, my name is John Smith.";
 
         DeidentificationContent content = new DeidentificationContent(inputText);
-        content.setOperation(OperationType.SURROGATE);
-        content.setDataType(DocumentDataType.PLAINTEXT);
 
-        DeidentificationResult result = deidentificationClient.deidentify(content).block();
```

Comment:
nit, using `block()` (or `getSyncPoller` above) in Async sample kind of defeat the purpose of Async...

Typically, it should use `.subscribe`, e.g.
https://github.com/Azure/azure-sdk-for-java/blob/2e191d19f153ff1c989e1bad8cc432d152e35c50/sdk/healthdataaiservices/azure-health-deidentification/src/samples/java/com/azure/health/deidentification/AsyncListCompletedFiles.java#L48-L50

Reply:
Removed file